common/changes/@microsoft/rush/chore-rush-plugin-schema_2021-12-01-05-41.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "set $schema in rush plugin related json files",
```

```suggestion
      "comment": "",
```

The changelog for the base feature will cover this.